spec/acceptance/docker_full_spec.rb

```diff
@@ -0,0 +1,643 @@
+require 'spec_helper_acceptance'
+
+describe 'Temp OS hacks' do
```

Comment:
One of these installs docker and the other doesn't? I think this assumes the other test cases are run on the same machine as these? This should be a before block on a global context if that's the intent. Otherwise (I think) other tests will fail if you run them individually.